### PR TITLE
Prisma token renewal

### PR DESCRIPTION
**Pull request by paskal**
Renew the Prisma token not more often than once in 3 minutes, re-login in case of problems.

### CODE REVIEW

File: api/prisma.go [hunk 1]
```diff
@@ -90,13 +75,18 @@ func (p *Prisma) GetAPIHealthStatus() int {
...
 
-// doAPIRequest does request to API with specified method and returns response body on success, thread-safe
+// doAPIRequest does request to API with specified method and returns response body on success
 func (p *Prisma) doAPIRequest(method, url string, body io.Reader) ([]byte, error) {
 	req, err := http.NewRequest(method, p.APIUrl+url, body)
 	if err != nil {
 		return nil, errors.Wrap(err, "error creating request")
 	}
 	req.Header.Set("Content-Type", "application/json")
+	if time.Since(p.tokenRenewTime) > time.Minute*3 {
```
Comment (kostty) [inline api/prisma.go]: is 3 a "magic number" here?

File: main.go [hunk 2]
```diff
@@ -91,16 +91,13 @@ func main() {
...
 func prepareCollectors(opts opts) (*collectors, error) {
 	var collectors = &collectors{}
-	var err error
 	if opts.PrismAPIKey != "" && opts.PrismAPIPassword != "" {
-		log.Print("[INFO] Initialising Prisma data collection")
-		collectors.prisma, err = api.NewPrisma(opts.PrismAPIKey, opts.PrismAPIPassword, opts.PrismAPIUrl)
-		if err != nil {
-			return nil, errors.Wrap(err, "can't connect to Prisma")
-		}
+		log.Printf("[INFO] Initialising Prisma data collection with API key %s", opts.PrismAPIKey)
```
Comment (thecrogers) [inline main.go:95]: why are you logging the api key?

**Review by thecrogers — APPROVED** [review]
LGTM

File: main.go (hunk 2 above)
Comment (paskal) [reply main.go:95]: It's login and would help troubleshoot the issues with authentification. The key is useless without a password.

**Pull request merged**